models/azure/managed_identity.go

```diff
@@ -19,20 +19,20 @@ package azure
 
 import "github.com/bloodhoundad/azurehound/enums"
 
-// Identity for the virtual machine.
-type VirtualMachineIdentity struct {
-	// The principal id of the virtual machine identity. The property will only be provided for a system assigned
+// Managed identity.
+type ManagedIdentity struct {
```

Comment:
Not sure what you mean by breaking, since I've changed it in the VirtualMachine model as well. I've added the definition, however, nothing is using it at the moment. Do you want to keep using it for the VM model?

Reply:
So `github.com/bloodhoundad/azurehound` can be imported as a library in addition to being a command-line program. While I don't know of anyone outside of our organization that is using it as a library I do know it's something that folks are doing internal to our organization. So while the `azurehound` binary may not be using it anymore I want to limit impacting downstream users when possible.

Reply:
Since they share the same type definition and only differ by their type name I think it's fine to keep the change you made to the VM model.

Reply:
added for compatibility